src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/listeners/ReferralMetricsListener.kt

```diff
@@ -34,16 +34,14 @@ class ReferralMetricsListener(
   }
 
   private fun timer(previous: Referral, current: Referral): Timer {
-    return if (current.intervention.id == previous.intervention.id) {
-      Timer.builder("intervention.referral.repeat_time")
-        .tag("crn", current.serviceUserCRN)
-        .tag("interventionId", current.intervention.id.toString())
-        .register(registry)
-    } else {
-      Timer.builder("intervention.referral.redirect_time")
-        .tag("crn", current.serviceUserCRN)
-        .register(registry)
-    }
+    val interventionKind =
+      if (current.intervention.id == previous.intervention.id) "same"
+      else "different"
+
+    return Timer.builder("intervention.referral.repeated_time")
+      .tag("crn", current.serviceUserCRN)
+      .tag("intervention", interventionKind)
```

Comment:
You removed the interventionId, will that not be useful to capture that as well?

Reply:
I realised I am not using it in any of the graphs, only the fact that something is "same" or "different" intervention. So it's a kind of YAGNI

I figured if we want to dig into what exactly happened, we could look up a CRN's referral history

Does that make sense? Do you agree?

Reply:
@sldblog I agree with your point.